src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -68,10 +76,6 @@ private void initializeDrivingVariants() {
         //Note: we are disabling lookahead here because of windowed queries that need to "look behind" as well.
         drivingVariantsFeatureInput = new FeatureInput<>("drivingVariantFile", Collections.emptyMap(), drivingVariantFile);
         features.addToFeatureSources(0, drivingVariantsFeatureInput, VariantContext.class);
-
-        if ( hasIntervals() ) {
-            drivingVariants.setIntervalsForTraversal(intervalsForTraversal);
-        }
     }
```

Comment:
Add a comment here mentioning that the intervals for the driving variants are now set in `postInitialization()`